testing/bdd/features/timing/computed_times.feature

```diff
@@ -56,7 +56,14 @@ Feature: Computed times computation
     | clock     |             |              |          |             |             |            |            | 00:00:15.0  | 00:00:30.0  | 00:00:10.0  | 00:00:40.0 |             |           | 00:00:10.0     | 00:00:40.0   | 00:00:00.0  |
     | clock     |             |              |          |             |             |            |            | 00:00:15.0  | 00:00:30.0  | 00:00:10.0  | 00:00:40.0 | 00:00:05.0  | 00:00:14.0| 00:00:05.0     | 00:00:40.0   | 00:00:00.0  |
     | clock     |             |              |          |             |             |            |            | 00:00:15.0  | 00:00:30.0  | 00:00:05.0  | 00:00:40.0 | 00:00:10.0  | 00:00:14.0| 00:00:05.0     | 00:00:40.0   | 00:00:00.0  |
-    @skip
+
+    | clock     | 00:00:10.0  | 00:00:05.0   |          |             |             |            |            |             |             |             |            |             |           | 00:00:00.0     |              | 00:00:00.0  |
+    | clock     |             |              |          | 00:00:10.0  | 00:00:05.0  |            |            |             |             |             |            |             |           | 00:00:00.0     |              | 00:00:00.0  |
+    | clock     |             |              |          |             |             | 00:00:10.0 | 00:00:05.0 |             |             |             |            |             |           | 00:00:00.0     |              | 00:00:00.0  |
+    | clock     |             |              |          |             |             |            |            | 00:00:10.0  | 00:00:05.0  |             |            |             |           | 00:00:00.0     |              | 00:00:00.0  |
+    | clock     |             |              |          |             |             |            |            | 00:00:10.0  | 00:00:05.0  | 00:00:15.0  | 00:00:10.0 |             |           | 00:00:00.0     |              | 00:00:00.0  |
+    | clock     |             |              |          |             |             |            |            | 00:00:10.0  | 00:00:05.0  | 00:00:15.0  | 00:00:10.0 | 00:00:05.0  | 00:00:01.0| 00:00:00.0     |              | 00:00:00.0  |
```

Comment:
Should duplicate the above scenarios setting all the elements to have begin == end; should have the same result.